From 81a71ae73cc6c60101fb583b87b5ba0085e164bb Mon Sep 17 00:00:00 2001
From: nbruin <nbruin@sfu.ca>
Date: Wed, 21 Feb 2024 20:20:46 -0800
Subject: [PATCH] Remove long/int relic

in py2, long and int were different types. In py3, only int remains, with long a synonym in cython.
---
 src/sage/rings/integer_ring.pyx | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

Index: sage-patch-test/sage/src/sage/rings/integer_ring.pyx
===================================================================
--- sage-patch-test.orig/sage/src/sage/rings/integer_ring.pyx
+++ sage-patch-test/sage/src/sage/rings/integer_ring.pyx
@@ -588,9 +588,7 @@ cdef class IntegerRing_class(PrincipalId
             ...
             TypeError: must be a Python int object
         """
-        if S is long:
-            return sage.rings.integer.long_to_Z()
-        elif S is int:
+        if S is int:
             return sage.rings.integer.int_to_Z()
         elif S is bool:
             return True
